samples/client/petstore/swift3/default/PetstoreClient/Classes/Swaggers/APIHelper.swift

```diff
@@ -48,32 +46,13 @@ class APIHelper {
         }
         return destination
     }
-
+    
     static func mapValuesToQueryItems(values: [String:Any?]) -> [URLQueryItem]? {
-        let returnValues = values
-            .filter { $0.1 != nil }
-            .map { (item: (_key: String, _value: Any?)) -> [URLQueryItem] in
-                if let value = item._value as? Array<String> {
-                    return value.map { (v) -> URLQueryItem in
-                        URLQueryItem(
-                            name: item._key,
-                            value: v
-                        )
-                    }
-                } else {
-                    return [URLQueryItem(
-                        name: item._key,
-                        value: "\(item._value!)"
-                    )]
-                }
-            }
-            .flatMap { $0 }
-
-        if returnValues.count == 0 {
-            return nil
+        let returnValues = values.flatMap { (item: (key: String, value: Any?)) -> URLQueryItem? in
+            guard let value = item.value else { return nil }
+            return URLQueryItem(name: item.key, value:"\(value)")
```

Comment:
If you could isolate this PR to a specific bugfix, that would be a start. I would be happy to review style changes separate from a bugfix as well.